pkg/k8s/watchers/pod.go

```diff
@@ -263,10 +270,19 @@ func (k *K8sWatcher) updateK8sPodV1(oldK8sPod, newK8sPod *slim_corev1.Pod) error
 		return k.deleteK8sPodV1(newK8sPod)
 	}
 
-	// The pod IP can never change, it can only switch from unassigned to
-	// assigned
-	// Process IP updates
-	k.addK8sPodV1(newK8sPod)
+	if newK8sPod.Spec.HostNetwork {
```

Comment:
Why is this check required? AFAIK, k8s doesn't let you update pod spec in order to switch to `hostNetwork`.

Reply:
We need to ignore `HostNetwork` pod events. This is the only place we can ignore them as far as I can see.